lib/datasets/estatjp.rb

```diff
@@ -0,0 +1,302 @@
+# frozen_string_literal: true
+
+require 'digest/md5'
+require 'net/http'
+require 'uri'
+require 'json'
+
+module Datasets
+  Record = Struct.new(:id, :name, :values)
+
+  # Estat module
+  module Estatjp
```

Comment:
We used `XXXJapan` naming in `lib/datasets/postal-code-japan.rb`.
How about `EStatJapan`? (It'll be better that we use `S` instead of `s` because `e-Stat` uses `S`.)

Reply:
I agree with you. I'll rename to `EStatJapan` as you suggested.